lib/openshift-rest-client.js

```diff
@@ -42,7 +42,11 @@ const resourceAliases = {
   'user.openshift.io': ['user']
 };
 
-// function is passed in and called by the kubernetes-client to add the openshift aliases
+/**
+ * Function is passed in and called by the kubernetes-client to add the openshift aliases.
+ * @param {string} resourceType alias array element.
```

Comment:
I would say we don't need docs here since this is an internal method

Reply:
Makes sense, thanks for the review, I'm going to remove the docs. :+1: